### PR TITLE
Upgrade chart version to v5.0.3 for CVE-2019-11255

### PR DESCRIPTION
**Pull request by pure-yesmat**

### CODE REVIEW

File: operator-csi-plugin/README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ The ``update.sh`` script is used to apply changes from ``values.yaml`` as follow
 ## Uninstall
 To uninstall the Pure CSI Operator, run 
 ```
+kubectl delete PSOPlugin/psoplugin-operator
```
Comment (sdodsley) [inline operator-csi-plugin/README.md]: Does this not happen anyway with the `delete all`?

File: operator-k8s-plugin/README.md [hunk 2]
```diff
@@ -108,6 +108,7 @@ More details on using configuration labels can be found [here](../docs/flex-volu
 ## Uninstall
 To uninstall the Pure FlexVolume Operator, run
 ```
+kubectl delete PSOPlugin/psoplugin-operator
```
Comment (sdodsley) [inline operator-k8s-plugin/README.md]: Same as above...

File: docs/index.yaml [hunk 3]
```diff
@@ -18,7 +33,7 @@ entries:
     version: 1.0.2
   - apiVersion: v1
     appVersion: 1.0.1
```
Comment (sdodsley) [inline docs/index.yaml]: Still 1.0.1 or should it be 1.0.3?

File: docs/index.yaml [hunk 4]
```diff
@@ -18,7 +33,7 @@ entries:
     version: 1.0.2
```
Comment (sdodsley) [inline docs/index.yaml]: As above

File: docs/index.yaml [hunk 5]
```diff
@@ -1,9 +1,24 @@
...
+    - purestorage
+    maintainers:
+    - email: sales@purestorage.com
+      name: Pure Storage, Inc.
+    name: pure-csi
+    urls:
+    - https://purestorage.github.io/helm-charts/pure-csi-1.0.3.tgz
+    version: 1.0.3
   - apiVersion: v1
     appVersion: 1.0.2
```
Comment (sdodsley) [inline docs/index.yaml]: Does this need to be here, or even changed to 1.0.3?

File: docs/index.yaml [hunk 6]
```diff
@@ -18,7 +33,7 @@ entries:
     version: 1.0.2
   - apiVersion: v1
```
Comment (sdodsley) [inline docs/index.yaml]: Maybe its the format in GitHub but there seem to be a lot of references to appversion and version

File: operator-csi-plugin/README.md (hunk 1 above)
Comment (pure-yesmat) [reply operator-csi-plugin/README.md]: order matters, if you delete the children of the CRD it gets stuck deleting the CRD for some reason.

File: docs/index.yaml (hunk 6 above)
Comment (pure-yesmat) [reply docs/index.yaml]: I don't follow

File: operator-csi-plugin/README.md (hunk 1 above)
Comment (pure-yesmat) [reply operator-csi-plugin/README.md]: although I think this is missing the namespace.

File: operator-csi-plugin/README.md [hunk 7]
```diff
@@ -122,10 +123,6 @@ To completely remove the CustomResourceDefinition used by the Operator run
 ```
 kubectl delete crd psoplugins.purestorage.com
 ```
-If the CRD fails to delete you may be experiencing a known issue. Resolve this by running:
```
Comment (Pure-AdamuKaapan) [inline operator-csi-plugin/README.md]: Is this fixed by the extra `kubectl delete` at the top?

File: docs/index.yaml (hunk 6 above)
Comment (Pure-AdamuKaapan) [reply docs/index.yaml]: I think I see what @sdodsley is saying, it confused me too for a bit, but this looks correct. It looks like the helm chart has a list of all previous versions in it as well.

File: docs/index.yaml [hunk 8]
```diff
@@ -18,7 +33,7 @@ entries:
...
     appVersion: 1.0.1
-    created: "2019-11-11T10:13:06.2762877-08:00"
+    created: 2019-11-15T15:56:15.0235638-08:00
```
Comment (Pure-AdamuKaapan) [inline docs/index.yaml]: Did these change because you had to backport the fix or something similar?

File: docs/index.yaml (hunk 5 above)
Comment (Pure-AdamuKaapan) [reply docs/index.yaml]: I think this is part of the listing of past versions (see my comment response further down).

File: docs/index.yaml (hunk 6 above)
Comment (dsupure) [reply docs/index.yaml]: @Pure-AdamuKaapan is correct. Helm chart list all previous versions.

File: operator-csi-plugin/README.md (hunk 7 above)
Comment (pure-yesmat) [reply operator-csi-plugin/README.md]: yep

File: docs/index.yaml (hunk 8 above)
Comment (pure-yesmat) [reply docs/index.yaml]: I actually have no idea how that changed!

Comment (pure-yesmat) [reply docs/index.yaml]: This is automatically updated by helm. Looking at all our previous commits that seems to have always been the case!

**Comment by pure-yesmat** [comment]
Would love to get some more eyes on this.

**Pull request merged**